sklearn/linear_model/logistic.py

```diff
@@ -1178,6 +1178,13 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
     Attributes
     ----------
 
+    classes_ : array or shape (n_classes, )
+        A list of class labels known to the classifier.
+
+        In binary classification, one class is usually considered the
```

Comment:
Why do you add this paragraph? I don't think we need it here.

Reply:
I can remove this entire sentence:
>        In binary classification, one class is usually considered the
>        'positive' class. For example, in labels [-1, 1], 1 is the positive
>        class.